compute_tools/src/catalog.rs

```diff
@@ -140,5 +140,34 @@ pub async fn get_database_schema(
             warn!("pg_dump stderr: {}", line)
         }
     });
-    Ok(initial_stream.chain(stdout_reader.map(|res| res.map(|b| b.freeze()))))
+
```

Comment:
Haven't reviewed the actual fix, but let's add a check into `test_compute_catalog` that there are always ending lines in the dump?
https://github.com/neondatabase/neon/blob/4b2f56862dc5738407893df451348e78696abd65/test_runner/regress/test_compute_catalog.py#L83

I.e.
```
-- PostgreSQL database dump complete
```

Not sure it will ever fail in CI, but anyway